backend/nestjs/apps/listing-service/src/rooms/entities/room.entity.ts

```diff
@@ -0,0 +1,38 @@
+import { IsNotEmpty, IsUUID } from 'class-validator';
+import { AbstractEntity } from 'libs/database/src';
+import { Column, Entity, JoinColumn, ManyToOne } from 'typeorm';
+import { HotelEntity } from '../../hotels/entities/hotel.entity';
+
+@Entity({ name: 'rooms' })
+export class RoomEntity extends AbstractEntity<RoomEntity> {
+  @Column()
+  room_name: string;
+
+  @Column('text')
+  description: string;
+
+  @Column()
+  geolocation: string;
+
+  @Column()
+  main_pic: string;
```

Comment:
the same here

Reply:
@panoti I think property should be consistent the column field in the database.